core/src/main/java/bio/terra/pearl/core/service/notification/email/SendgridClient.java

```diff
@@ -38,6 +44,29 @@ public void sendEmail(Mail mail) throws Exception {
     sg.api(request);
```

Comment:
I looked into what it would take to tie these notifications to the actual events (opens, clicks, etc) that we also record in the new `sendgrid_event` table. It's possible, but there are some things worth noting related to how SendGrid handles Message IDs. See https://docs.sendgrid.com/glossary/what-is-x-message-id and https://docs.sendgrid.com/glossary/message-id

It most likely wouldn't be a lot of work to deal with this, and it could provide some very cool functionality in the admin UI that would allow study admins to monitor if participants are opening their emails or not, but it felt like scope creep for this ticket.